canonical/ubuntu.com
# Update forms to include country dial codes dropdown

## Done

- Implements country codes on modal forms

## QA

- Go to https://ubuntu-com-12025.demos.haus/ and open any modal form
- Select a country code ('uk' for United Kingdom) and input a phone number (ex. 07123456789)
- Submiit the form and check the phone number appears in the payload correctly formatted (+447123456789)
- Repeat without selecting a country code and input the number with the country code (ex. +447123456789)
- Check that the country code updates automatically and the payload is correct
- Check the country code can be accessed only using the keyboard
- Repeat this process with a couple modal forms and a couple static forms

Documentation for the plugin used can be found [here](https://github.com/jackocnr/intl-tel-input)
## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5844
